lib/web_ui/test/golden_files/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Failed screenshot files
```

Comment:
I think we should ignore the test output artifacts so people don't add them by mistake to the repo. Or maybe we can have a `felt goldens:clean` command that removes all the junk that failing (or almost-failing) tests generate.

Reply:
(Other than that, LGTM!)

Reply:
I thought my change stopped writing to the `test/` directory, but let me double-check.

Reply:
You're right, it's writing to test_results within the dart tool. DISREGARD!